pywps/configuration.py

```diff
@@ -17,6 +17,8 @@
 
 __author__ = "Calin Ciociu"
 
+from pywps.uri import file_uri
```

Comment:
maybe we can have `pywps.util` with these helper functions?

Reply:
good idea, I'll do that

Reply:
> maybe we can have `pywps.util` with these helper functions?

done